Z2A SwiftUI/NSCoffee/NSCoffee/Views/DrinkTableRow.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  DrinkTableRow.swift
+//  NSCoffee
+//
+//  Created by Rob Whitaker on 02/03/2025.
+//
+
+import SwiftUI
+
+struct DrinkTableRow: View {
+    let drink: any Drink
+
+    var body: some View {
+        HStack {
+            DrinkTableImage(imageName: drink.imageName)
+                .padding(.trailing, 10)
+
+            VStack(alignment: .leading) {
+                Text(drink.name)
+                    .lineLimit(1)
+
+                Text(CurrencyFormatter.format(drink.basePrice))
+                    .font(.system(size: 17.0))
```

Comment:
I think I configured this to dark gray to so it has low contrast in dark mode.

Reply:
Added.